Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAccountProtectionLocalAdministratorPasswordSolutionPolicy/MSFT_IntuneAccountProtectionLocalAdministratorPasswordSolutionPolicy.psm1

```diff
@@ -0,0 +1,1250 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $Identity,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.String]
+        $Description,
+
+        [Parameter()]
+        [System.Int32]
+        [ValidateSet(0, 1, 2)]
+        $BackupDirectory,
+
+        [Parameter()]
+        [System.Int32]
+        [ValidateScript({ $_ -ge 7 -and $_ -le 365 })]
```

Comment:
I think a ValidateRange is better suitable here:
https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/validaterange-attribute-declaration